DEVELOPING.md

```diff
@@ -592,6 +592,12 @@ mvn clean install -Pdist
 docker build -f contrib/docker/Dockerfile . -t emissary
 ```
 
+For a lightweight emissary darker image the Dockerfile.alpine can be used. 
+This image is 1/3 of the size of the normal centOS 7 based image
+```
+docker build -f contrib/docker/Dockerfile.alpine . -t emissary-light
```

Comment:
Should include the following line to clarify that this step is still needed regardless of base Dockerfile.  (This just mirrors the instruction steps that use the original Dockerfile):
```suggestion
mvn clean install -Pdist
docker build -f contrib/docker/Dockerfile.alpine . -t emissary-light
```